includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -70,10 +70,10 @@ private function getLangWithFallback( $lang ) {
 			'media-kit' => 'http://www.wikia.com/mediakit',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "media-kit" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)